model_analyzer/config/generate/run_config_generator_factory.py

```diff
@@ -102,29 +110,43 @@ def _create_brute_run_config_generator(
     @staticmethod
     def _create_quick_plus_concurrency_sweep_run_config_generator(
         command_config: ConfigCommandProfile, gpus: List[GPUDevice],
-        models: List[ModelProfileSpec], client: TritonClient,
-        result_manager: ResultManager,
+        models: List[ModelProfileSpec],
+        ensemble_submodels: Dict[str, List[ModelProfileSpec]],
+        client: TritonClient, result_manager: ResultManager,
         model_variant_name_manager: ModelVariantNameManager
     ) -> ConfigGeneratorInterface:
-        search_config = RunConfigGeneratorFactory._create_search_config(models)
+        search_config = RunConfigGeneratorFactory._create_search_config(
+            models, ensemble_submodels)
         return QuickPlusConcurrencySweepRunConfigGenerator(
             search_config=search_config,
             config=command_config,
             gpus=gpus,
             models=models,
+            ensemble_submodels=ensemble_submodels,
             client=client,
             result_manager=result_manager,
             model_variant_name_manager=model_variant_name_manager)
 
     @staticmethod
-    def _create_search_config(models: List[ModelProfileSpec]) -> SearchConfig:
+    def _create_search_config(
+            models: List[ModelProfileSpec],
+            ensemble_submodels: Dict[str,
+                                     List[ModelProfileSpec]]) -> SearchConfig:
         dimensions = SearchDimensions()
 
-        #yapf: disable
-        for i, model in enumerate(models):
-            dims = RunConfigGeneratorFactory._get_dimensions_for_model(model.supports_batching())
-            dimensions.add_dimensions(i, dims)
-        #yapf: enable
+        index = 0
+        for model in models:
```

Comment:
at this point, any ensemble models have been removed, and instead we have submodels (and/or normal models)?

Reply:
Correct.